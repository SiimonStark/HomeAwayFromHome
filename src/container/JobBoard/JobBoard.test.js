import React from 'react';
import {Jobboard} from './index';
import {shallow} from 'enzyme';
// import {connect} from 'react-redux';

describe('Jobboard', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Jobboard />);
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
})