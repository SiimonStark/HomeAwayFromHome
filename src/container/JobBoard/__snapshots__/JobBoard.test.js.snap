// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jobboard should match snapshot 1`] = `
<section
  className="Job__display"
>
  <div
    className="Jobboard__container"
  >
    <p>
      Test Jobboard
    </p>
  </div>
</section>
`;
